src/main/java/bio/terra/cli/service/utils/WorkspaceManagerService.java

```diff
@@ -230,6 +235,26 @@ public void deleteWorkspace(UUID workspaceId) {
     }
   }
 
+  /**
+   * Call the Workspace Manager PATCH "/api/workspaces/v1/{id}" endpoint to update an existing
+   * workspace.
+   *
+   * @param workspaceId the id of the workspace to update
+   * @return the Workspace Manager workspace description object
+   */
+  public WorkspaceDescription updateWorkspace(
+      UUID workspaceId, String displayName, String description) {
+    WorkspaceApi workspaceApi = new WorkspaceApi(apiClient);
+    try {
+      // update the Terra workspace object
+      UpdateWorkspaceRequestBody updateRequest =
```

Comment:
this will override an existing name or description with null if it's not re-specified. I guess that's the way to do it if you don't have a separate clearDisplayName command.